tests/src/test/java/alluxio/security/ClusterInitializationTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.security;
+
+import alluxio.AlluxioURI;
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.LocalAlluxioClusterResource;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+
+import alluxio.exception.ExceptionMessage;
+import alluxio.master.MasterTestUtils;
+import alluxio.master.file.FileSystemMaster;
+
+import alluxio.security.authentication.AuthenticatedClientUser;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+
+/**
+ * Unit tests for starting a cluster when security is enabled.
+ */
+public class ClusterInitializationTest {
+  @Rule
+  public ExpectedException mThrown = ExpectedException.none();
+
+  private static final String SUPER_USER = "alluxio";
+  private static final String USER = "jack";
+
+  private static final AlluxioURI ROOT = new AlluxioURI("/");
+
+  @Rule
+  public LocalAlluxioClusterResource mLocalAlluxioClusterResource =
+      new LocalAlluxioClusterResource(
+          LocalAlluxioClusterResource.DEFAULT_WORKER_CAPACITY_BYTES,
+          LocalAlluxioClusterResource.DEFAULT_USER_BLOCK_SIZE,
+          Constants.SECURITY_AUTHENTICATION_TYPE, "SIMPLE",
+          Constants.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true");
+
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {Constants.SECURITY_LOGIN_USERNAME, SUPER_USER})
+  public void startClusterTest() throws Exception {
```

Comment:
Sure, added some comments in the methods to describe its purpose. Since these methods are test methods, do we need javadoc? I could update another patch later if we want.